libvast/vast/system/application.hpp

```diff
@@ -46,5 +45,8 @@ class application {
   command root;
 };
 
-} // namespace vast::system
+/// Format a useful human readable error message on the output stream
+void render_error(const application& app, const caf::error& err,
+                  std::ostream& os);
```

Comment:
Also documentation tweaks needed here. Also, we use third person singular to begin a the first line of documentation.